### PR TITLE
Milestone 6: preserve data

### PR DESCRIPTION
**Pull request by medaminedev66**
- On exit from the app all the data (books, people, and rentals) is saved into JSON files (`rentals.json`, `peoole.json`, `books.json`).
- On the startup of the app, all data is loaded from files. if one or more of the files are missing the code will create a new file (or files) with an empty array.

### CODE REVIEW

File: test.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+person = Person.new(age: 10, name: 'amine')
+# person = {
+#     "name":
+#     "age":
+#     "permission":
+#     "specialization":
+#     "rentals":
+
+# }
```
Comment (o-rashad) [inline test.rb:20]: :bulb:

**Review by o-rashad — APPROVED** [review]
Asslam Alaikum Team, @medaminedev66  @julie-ify, 😊

Hi👋🏽, you naild it guys
Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding! 👏❤️️

## Note :
- [ ] _OPTINAL_ Would be nice to get rid of any extra comments in your codes, i commented on it with `:bulb:` emoji , Keep it up :100: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.rb (hunk 1 above)
Comment (medaminedev66) [reply test.rb:20]: 😮 I didn't realize that this file was pushed. I thought it was removed after writing the `git stash` command. I will remove it.

**Comment by medaminedev66** [comment]
Thanks, @od-c0d3r , for reviewing our pull request.

**Pull request merged**